compiler/noirc_frontend/src/parser/parser/function.rs

```diff
@@ -437,7 +451,7 @@ mod tests {
     fn parse_function_unclosed_parentheses() {
         let src = "fn foo(x: i32,";
         let (module, errors) = parse_program_with_dummy_file(src);
-        assert_eq!(errors.len(), 1);
+        assert!(!errors.is_empty());
```

Comment:
We are getting more than one error here now?

Reply:
Yes, the next error is the missing `{`. I guess we could try to improve this, though in Rust at least in the editor I also get two errors (on the command line I just get one). That said, once you add a parentheses you'll need `{` anyway, so maybe two errors is fine 🤷